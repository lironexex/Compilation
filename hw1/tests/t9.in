// strings 2
"\u{54}\u{69}\u{4D}\u{65}"
"\u{000057}\u{61}\u{053}\u{0074}\u{45}\u{0064}"
"\u{21}\t\u{22}\n\n\u{24}\r\n\u{2F}\u{3B}\\\\\u{3F}\u{3E}"
"\\u{21}\t\u{22}"
