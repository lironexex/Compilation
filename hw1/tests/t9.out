1 COMMENT 1
2 STRING TiMe
3 STRING WaStEd
4 STRING !	"

$
/;\\?>
5 STRING \u{21}	"
